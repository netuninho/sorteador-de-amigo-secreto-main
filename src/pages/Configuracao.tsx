import Footer from "../components/Footer/Footer"
import Formulario from "../components/Form/Formulario"
import ListaParticipantes from "../components/ListaParticipantes/ListaParticipantes"

const Configuracao = () => {
    return (
        <>
            <Formulario/>
            <ListaParticipantes/>
            <Footer/>
        </>
    )
}

export default Configuracao