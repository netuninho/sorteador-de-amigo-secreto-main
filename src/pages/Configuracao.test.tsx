import React from "react";
import { RecoilRoot } from "recoil";
import { render } from "@testing-library/react";
import Configuracao from "./Configuracao";

const mockNavigate = jest.fn()

jest.mock('react-router-dom', () => {
    return {
        useNavigate: () => mockNavigate
    }
})

describe('pagina de configuracao', () => {
    test('deve ser redenrizada corretamente', () => {
       const { container } = render(
            <RecoilRoot>
                <Configuracao/>
            </RecoilRoot>
        )

        expect(container).toMatchSnapshot()
    })
})