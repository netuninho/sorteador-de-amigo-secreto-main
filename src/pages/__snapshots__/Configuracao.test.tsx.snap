// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pagina de configuracao deve ser redenrizada corretamente 1`] = `
<div>
  <form
    class="sc-fQpRED cEQwWT"
  >
    <input
      class="sc-dsLQwm jJgtZl"
      placeholder="Insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      class="sc-iKTcqh bxVLOX"
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <footer
    class="sc-elDIKY dQFeAM"
  >
    <button
      class="sc-braxZu dcCZoM"
      disabled=""
    >
      <img
        alt="SÃ­mbolo de play"
        class="sc-gJhJTp eeSqeh"
        src="/img/play.png"
      />
      Iniciar brincadeira!
    </button>
    <img
      alt="sacolas"
      src="/img/sacolas.png"
    />
  </footer>
</div>
`;
